m365-developer-proxy/m365proxyrc.json

```diff
@@ -153,6 +153,11 @@
     "minMs": 200,
     "maxMs": 10000
   },
+  "rateLimiting": {
+    "costPerRequest": 1,
+    "rateLimit": 40,
```

Comment:
Let's align these values with defaults in the plugin

Reply:
Good idea. I took this config from the example in plugin page in the docs so I'll update that as well 👍

Reply:
@waldekmastykarz updated 👍